routes/web.php

```diff
@@ -11,6 +11,31 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
+
+
+
+Route::group(['as' => 'get-'], function () {
+
+	Route::get('/', ['as' => 'home-page', function () {
+	    
+		return App::make('App\Http\Controllers\UserController')->getData();
+	}]);
 });
+
+Route::group(['as' => 'post-'], function () {
```

Comment:
please use this convention:
Route::group(['prefix' => 'user']), function() {
	Route::post('delete', 'UserController@deleteData');
	Route::post('add', 'UserController@insertData');
	Route::post('edit', 'UserController@updateData');
});

Reply:
applied change